test/Driver/macabi-environment.swift

```diff
@@ -2,7 +2,7 @@
 
 // UNSUPPORTED: windows
 
-// RUN: %swiftc_driver -sdk "" -driver-print-jobs -target x86_64-apple-ios13.0-macabi -sdk %S/../Inputs/clang-importer-sdk %s | %FileCheck -check-prefix=IOS13-MACABI %s
+// RUN: %swiftc_driver -sdk "" -sdk "" -driver-print-jobs -target x86_64-apple-ios13.0-macabi -sdk %S/../Inputs/clang-importer-sdk %s | %FileCheck -check-prefix=IOS13-MACABI %s
```

Comment:
Not a problem, but in skimming through this, I noticed a few lines like this that have `-sdk "" -sdk""` twice.